src/main/java/seedu/address/logic/commands/VisitorInCommand.java

```diff
@@ -0,0 +1,120 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VISITOR;
+
+import java.util.Set;
+
+import javafx.collections.ObservableList;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Nric;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.visitor.Visitor;
+import seedu.address.model.visitor.VisitorList;
+
+
+
+
+/**
+ * @@ GAO JIAXIN
+ * Checks in a visitor into patient's visitor list.
+ */
+public class VisitorInCommand extends Command {
```

Comment:
Please adhere to the naming format for *Command classes; as the command word is 'visitorin', 'in' should not be capitalised in the command name.